packages/Webkul/Admin/src/Resources/views/marketing/promotions/cart-rules/create.blade.php

```diff
@@ -251,7 +254,8 @@ class="ltr:pr-10 rtl:pl-10"
                                     :key="index"
                                     :index="index"
                                     @onRemoveCondition="removeCondition($event)">
-                                />
+                                >
+                                </v-cart-rule-condition-item>
```

Comment:
Hello @claudioghirellifh, 
We don't have to make all the changes. Just add the closing tag here like this (/>) and remove the (>) from line no. 256.

Reply:
Hi @amit-webkul 
There really were some unnecessary changes. I hadn't realized they had gone along. I already made the correction. Thank you very much